src/Servers/IIS/AspNetCoreModuleV2/CommonLibTests/CommonLibTests.vcxproj

```diff
@@ -50,7 +49,7 @@
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="Configuration">
-    <VCToolsVersion />
+    <VCToolsVersion>14.29.30133</VCToolsVersion>
```

Comment:
Why are we adding these?

Reply:
The default we get from VS2022 is higher, its 14.30xx, so we need to go lower than current because we have to use the v142 toolset that matches runtimes, I can't remember which variant of the build it was that would fail without this, I think its the arm64 or x86, this wasn't necessary for x64, I added this later to get the other variants working